runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -242,15 +145,15 @@ public void onConfigChanged(ConfigChanged event)
 
 		if (event.getKey().equals("gameAlwaysOnTop"))
 		{
-			if (this.isAlwaysOnTopSupported())
+			if (frame.isAlwaysOnTopSupported())
 			{
-				this.setAlwaysOnTop(Boolean.valueOf(event.getNewValue()));
+				frame.setAlwaysOnTop(config.gameAlwaysOnTop());
```

Comment:
This entire method should probably be on the EDT

Reply:
EDT?

Reply:
Event dispatch thread, aka the GUI thread

Reply:
Moved